CreeDictionary/API/models.py

```diff
@@ -236,13 +238,30 @@ def save(self, *args, **kwargs):
 
         super(Wordform, self).save(*args, **kwargs)
 
-    @classmethod
-    def search(
-        cls, query: str, affix_search: bool = True, **constraints
-    ) -> SortedSet["SearchResult"]:
-        from .search import WordformSearch
+    @staticmethod
+    def search_with_affixes(query: str) -> SortedSet["SearchResult"]:
+        """
+        Search for wordforms matching:
+         - the wordform text
+         - the definition keyword text
+         - affixes of the wordform text
+         - affixes of the definition keyword text
+        """
+        from .search import WordformSearchWithAffixes
```

Comment:
Is this here because of the import cycle issue?

Reply:
Yes. Although now that I reflect on it, I think having static methods for `Wordform.simple_search()` and `Wordform.search_with_affixes()` is not worth the ugly import cycle issues, so perhaps we should move that elsewhere!